core/templates/dev/head/admin/admin.html

```diff
@@ -11,93 +11,249 @@
+                                  <span ng-if="numUnseenNotifications > 0">
+                                    (<[numUnseenNotifications]>)
+                                  </span>
+                                </a>
+                              </li>
+                              <li>
+                                <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="/preferences">Preferences</a>
+                              </li>
+
+                              {% if is_moderator %}
+                                <li>
+                                  <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="/moderator" target="_blank">
+                                    Moderator Page
+                                  </a>
+                                </li>
+                              {% endif %}
+                              {% if is_super_admin %}
+                                <li>
+                                  <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="/admin" target="_blank">
+                                    Admin Page
+                                  </a>
+                                </li>
+                              {% endif %}
+                              <hr class="oppia-top-right-menu-item-separator">
+                              <li>
+                                <a ng-click="onMouseoutProfilePictureOrDropdown($event)" href="{{logout_url}}">Logout</a>
+                              </li>
+                            </ul>
+                          </li>
+                        {% else %}
+                          <li class="dropdown oppia-clickable-dropdown pull-right">
+                            <div class="oppia-navbar-button-container" style="margin-right: 10px;">
+                              <button class="btn oppia-navbar-button"
+                                      ng-click="onLoginButtonClicked('{{login_url}}')">
+                                <span>Sign in</span>
+                                <span class="caret"></span>
+                              </button>
+                            </div>
+                            <ul class="dropdown-menu oppia-navbar-dropdown" role="menu" style="margin-right: 15px; padding: 0;" ng-mouseover="onMouseoverDropdownMenu($event)" ng-mouseleave="onMouseoutDropdownMenu($event)">
+                              <li>
+                                <a href style="padding: 0; width: 200px;" ng-click="onLoginButtonClicked('{{login_url}}')">
+                                  <img src="/images/signin/Red-signin-Long-base-44dp.png">
+                                </a>
+                              </li>
+                            </ul>
+                          </li>
+                        {% endif %}
+                        {% if nav_mode != 'create' and nav_mode != 'explore' %}
+                          <ul ng-if="windowIsNarrow" class="nav oppia-navbar-tabs-narrow">
```

Comment:
This clause is not necessary; we aren't showing different things if the window is wide/narrow, right?